advertising/transaction-message/reference.md

```diff
@@ -95,8 +95,12 @@ Defines the amenities that are part of a package deal.
 |Occupancy|Optional.<br />Data type is integer.<br /><br />The maximum number of guests that the room is intended to accommodate. For example, although the room is physically capable of hosting 4 guests (capacity), it's intended for only 2 guests.<br /><br />**NOTES:**<ul><li>Although optional, you're strongly encouraged to always specify this element.</li><li>The value must be in the range 1 through 20.</li></ul>|None
 |OccupancyDetails|Optional.<br /><br />A container object that contains details about occupancy. If you include `Occupancy`, you may also specify this element. |[OccupancyDetails Type](#occupancydetails-type)
 |ChargeCurrency|Optional<br />Data type is string.<br /><br />Defines when the user pays for a booking. The following are the possible case-insensitive values.<ul><li>Deposit&mdash;The user is charged a portion at booking and the remainder at a later point in time, typically when the user checks out of the hotel.</li><li>Hotel&mdash;The user is charged at check-in. If the payment must be made in the hotel's currency, use this option.</li><li>Installment&mdash;The user is charged a fixed amount at specific intervals over fixed period of time.</li><li>Web&mdash;The user is charged online at the time of booking.</li></ul>The default is web.|None
-|Refundable|Required.<br /><br />An element that determines whether the room charge is refundable.<br /><br />**Attributes**<ul><li>available &mdash; Required. Set to true (or 1) if the room charge is refundable; otherwise, false (or 0).</li><li>refundable_until_days &mdash; Optional. The date by which the guest must request a refund. Specify the date as the number of days in advance of check-in. If not specified, the room charge is refundable up to and including the check-in date. Valid values are 0 through 180.</li><li>refundable_until_time &mdash; Optional. The guest may request a refund up until this time on the date referenced in the *refundable_until_days* attribute. Specify the time in the hotel's local time. Specify the time in the form, hh:mm:ss. For example, if the refund is available until 4 PM, use 16:00:00.</li></ul>|None
+|Refundable|Required.<br /><br />An element that determines whether the room charge is **fully** refundable.<br /><br />**Attributes**<ul><li>available &mdash; Required. Set to true (or 1) if the room charge is refundable; otherwise, false (or 0).</li><li>refundable_until_days &mdash; Optional. The date by which the guest must request a refund. Specify the date as the number of days in advance of check-in. If not specified, the room charge is refundable up to and including the check-in date. Valid values are 0 through 180.</li><li>refundable_until_time &mdash; Optional. The guest may request a refund up until this time on the date referenced in the *refundable_until_days* attribute. Specify the time in the hotel's local time. Specify the time in the form, hh:mm:ss. For example, if the refund is available until 4 PM, use 16:00:00.</li></ul>|None
```

Comment:
The emphasis seems subjective to me. Just an editorial suggestion. Feel free to disregard.
```suggestion
|Refundable|Required.<br /><br />An element that determines whether the room charge is fully refundable.<br /><br />**Attributes**<ul><li>available &mdash; Required. Set to true (or 1) if the room charge is refundable; otherwise, false (or 0).</li><li>refundable_until_days &mdash; Optional. The date by which the guest must request a refund. Specify the date as the number of days in advance of check-in. If not specified, the room charge is refundable up to and including the check-in date. Valid values are 0 through 180.</li><li>refundable_until_time &mdash; Optional. The guest may request a refund up until this time on the date referenced in the *refundable_until_days* attribute. Specify the time in the hotel's local time. Specify the time in the form, hh:mm:ss. For example, if the refund is available until 4 PM, use 16:00:00.</li></ul>|None
```